plugins/unifying/lu-device-runtime.c

```diff
@@ -85,22 +86,19 @@ lu_device_runtime_open (LuDevice *device, GError **error)
 		}
 	}
 	if (release != 0xffff) {
-		g_autofree gchar *devid2 = NULL;
 		switch (release &= 0xff00) {
 		case 0x1200:
 			/* Nordic */
 			devid2 = g_strdup_printf ("USB\\VID_%04X&PID_%04X",
 						  (guint) LU_DEVICE_VID,
 						  (guint) LU_DEVICE_PID_BOOTLOADER_NORDIC);
-			lu_device_add_guid (device, devid2);
```

Comment:
We still want these original GUIDs, a device can have lots of GUIDs with no ill effect.